packages/ui/src/components/Organisms/ContentHub.tsx

```diff
@@ -70,24 +70,28 @@ export function ContentHub({ pageSize = 10 }: { pageSize: number }) {
   );
 }
 
-const Card = (item: ContentHubResultItemFragment) => {
-  const id = item.title.toLowerCase().replace(/\s+/g, '-');
-
+const Card = ({
+  item,
+  id,
+}: {
+  item: ContentHubResultItemFragment;
+  id: number;
+}) => {
   return (
     <article
-      aria-labelledby={id}
-      className="focus-within:outline focus-within:outline-2 focus-within:outline-offset-2 focus-within:outline-indigo-600 relative max-w-sm bg-white rounded-lg hover:shadow dark:bg-gray-800 dark:border-gray-700 overflow-hidden flex flex-col-reverse"
+      aria-labelledby={`${id}`}
```

Comment:
mmm `id` is already a JavaScript string, so `aria-labelledby={id}` should work, no need to parse into a template literal